### PR TITLE
Pin ArduinoJson to v6 (6.x branch)

### PR DESCRIPTION
**Pull request by tyeth**

### CODE REVIEW

File: .github/workflows/build-clang-doxy.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
         echo >>$GITHUB_ENV WS_VERSION=$(git describe --dirty --tags)
     - uses: actions/checkout@v4
       with:
-         repository: adafruit/ci-arduino
+         repository: tyeth/ci-arduino
```
Comment (brentru) [inline .github/workflows/build-clang-doxy.yml:39]: @tyeth  Please change back to adafruit/ then OK to PR and merge in

File: .github/workflows/build-clang-doxy.yml [hunk 2]
```diff
@@ -106,7 +107,7 @@ jobs:
         echo >>$GITHUB_ENV WS_VERSION=$(git describe --dirty --tags)
     - uses: actions/checkout@v4
       with:
-        repository: adafruit/ci-arduino
+        repository: tyeth/ci-arduino
```
Comment (brentru) [inline .github/workflows/build-clang-doxy.yml:110]: here too

File: .github/workflows/build-clang-doxy.yml [hunk 3]
```diff
@@ -205,13 +207,14 @@ jobs:
         echo >>$GITHUB_ENV WS_VERSION=$(git describe --dirty --tags)
     - uses: actions/checkout@v4
       with:
-         repository: adafruit/ci-arduino
+         repository: tyeth/ci-arduino
```
Comment (brentru) [inline .github/workflows/build-clang-doxy.yml:210]: here too

File: .github/workflows/build-clang-doxy.yml [hunk 4]
```diff
@@ -256,13 +259,14 @@ jobs:
...
```
Comment (brentru) [inline .github/workflows/build-clang-doxy.yml:262]: here too

File: .github/workflows/build-clang-doxy.yml [hunk 5]
```diff
@@ -306,12 +310,13 @@ jobs:
...
```
Comment (brentru) [inline .github/workflows/build-clang-doxy.yml:313]: here too

File: .github/workflows/build-clang-doxy.yml [hunk 6]
```diff
@@ -346,12 +351,13 @@ jobs:
...
```
Comment (brentru) [inline .github/workflows/build-clang-doxy.yml:354]: here too

File: .github/workflows/build-clang-doxy.yml [hunk 7]
```diff
@@ -464,7 +470,7 @@ jobs:
 
     - uses: actions/checkout@v4
       with:
-         repository: adafruit/ci-arduino
+         repository: tyeth/ci-arduino
```
Comment (brentru) [inline .github/workflows/build-clang-doxy.yml:473]: here too

File: .github/workflows/build-clang-doxy.yml [hunk 8]
```diff
@@ -404,12 +409,13 @@ jobs:
         echo >>$GITHUB_ENV WS_VERSION=$(git describe --dirty --tags)
     - uses: actions/checkout@v4
       with:
-         repository: adafruit/ci-arduino
+         repository: tyeth/ci-arduino
```
Comment (brentru) [inline .github/workflows/build-clang-doxy.yml:412]: here too

File: .github/workflows/build-clang-doxy.yml [hunk 9]
```diff
@@ -390,8 +396,7 @@ jobs:
       fail-fast: false
       matrix:
         arduino-platform: ["feather_esp32s2_debug", "feather_esp32s2_tft_debug",
-                           "feather_esp32s3_debug", "feather_esp32s3_4mbflash_2mbpsram_debug", "feather_esp32s3_tft_debug",
-                           "feather_esp32s3_tft_debug"]
+                           "feather_esp32s3_debug", "feather_esp32s3_4mbflash_2mbpsram_debug", "feather_esp32s3_tft_debug"]
```
Comment (brentru) [inline .github/workflows/build-clang-doxy.yml:399]: ensure this matches `main` please

**Review by brentru — CHANGES_REQUESTED** [review]

**Review by brentru — APPROVED** [review]

File: .github/workflows/build-clang-doxy.yml (hunk 9 above)
Comment (tyeth) [reply .github/workflows/build-clang-doxy.yml:399]: leaving this change after discussion

**Pull request merged**